### PR TITLE
Creates the Kendra plugin section in Compare Search Results

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Creates the Kendra plugin section in Compare Search Results.

This is targeted to be released when the Kendra plugin is released (currently, scheduled for 1/9/23. The previous PR was #2012 and has been already reviewed.

### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by macohen — APPROVED** [review]
Please check links after the external docs are released. Thanks!

File: _search-plugins/search-relevance/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Alternatively, you can add sample data in OpenSearch Dashboards using the follow
 1. Select **Add sample data**.  
 1. Choose one of the built-in datasets and select **Add data**.
 
-## Using search relevance in OpenSearch Dashboards
+## Using Compare Search Results in OpenSearch Dashboards
```
Comment (Naarcha-AWS) [inline _search-plugins/search-relevance/index.md:35]: Is "Compare Search Results" a UI element? If not, we shouldn't capitalize it.

```suggestion
## Using compare search results in OpenSearch Dashboards
```

File: _search-plugins/search-relevance/index.md [hunk 2]
```diff
@@ -98,4 +98,60 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
 {: .note}
 
 You cannot save a given comparison for future use, so Compare Search Results is not suitable for systematic testing.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Comparing OpenSearch search results with re-ranked results
+
+One use case of Compare Search Results is to compare raw OpenSearch results with the same results processed by a re-ranking application. An example of such a re-ranker is **Kendra Intelligent Ranking for OpenSearch**, contributed by the Amazon Kendra team. This plugin takes search results from OpenSearch and applies Kendra’s semantic relevance rankings calculated using vector embeddings and other semantic search techniques. For many applications, this gives better result rankings.
```
Comment (Naarcha-AWS) [inline _search-plugins/search-relevance/index.md:105]:
```suggestion
One use case of compare search results is to compare raw OpenSearch results with the same results processed by a re-ranking application. An example of such a re-ranker is **Kendra Intelligent Ranking for OpenSearch**, contributed by the Amazon Kendra team. This plugin takes search results from OpenSearch and applies Kendra’s semantic relevance rankings calculated using vector embeddings and other semantic search techniques. For many applications, this gives better result rankings.
```

**Review by Naarcha-AWS — APPROVED** [review]
A couple of small changes, but otherwise looks good.

**Review by carolxob — APPROVED** [review]
LGTM with Nate's comments.

File: _search-plugins/search-relevance/index.md (hunk 1 above)
Comment (kolchfa-aws) [reply _search-plugins/search-relevance/index.md:35]: Compare Search Results is the name of the feature, and the name of the UI page. As per editorial review of the previous PR, we are capitalizing this feature. @natebower please confirm. Thank you!

File: _search-plugins/search-relevance/index.md (hunk 2 above)
Comment (natebower) [inline _search-plugins/search-relevance/index.md:105]:
```suggestion
One use case for Compare Search Results is to compare raw OpenSearch results with the same results processed by a re-ranking application. An example of such a re-ranker is **Kendra Intelligent Ranking**, contributed by the Amazon Kendra team. This plugin takes search results from OpenSearch and applies Amazon Kendra’s semantic relevance rankings calculated using vector embeddings and other semantic search techniques. For many applications, this provides better result rankings.
```

File: _search-plugins/search-relevance/index.md [hunk 3]
```diff
@@ -98,4 +98,60 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
...
+
+To try Kendra Intelligent Ranking, you must first set up the Amazon Kendra service. To get started, see [Amazon Kendra](https://aws.amazon.com/kendra/). For detailed information, including plugin setup instructions, see [Intelligently ranking OpenSearch (self managed) results using Amazon Kendra](https://docs.aws.amazon.com/kendra/latest/dg/opensearch-rerank.html).
+
+Once you've set up Kendra Intelligent Ranking, enter a query in **Query 1** and the same query using the Kendra Intelligent Ranking service for re-ranking in **Query 2**. Then compare search results from OpenSearch and Amazon Kendra.
```
Comment (natebower) [inline _search-plugins/search-relevance/index.md:109]:
```suggestion
Once you've set up Kendra Intelligent Ranking, enter a query in **Query 1** and enter the same query using the Kendra Intelligent Ranking service for re-ranking in **Query 2**. Then compare the search results from OpenSearch and Amazon Kendra.
```

Comment (natebower) [reply _search-plugins/search-relevance/index.md:109]: Can this read "...and enter the same query using Kendra Intelligent Ranking in **Query 2**."?

File: _search-plugins/search-relevance/index.md [hunk 4]
```diff
@@ -98,4 +98,60 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
...
+
+### Example
+
+The following example searches for the text "snacking nuts" in the `abo` index. The documents in the index hold snack descriptions in the `bullet_point` array.
```
Comment (natebower) [inline _search-plugins/search-relevance/index.md:113]:
```suggestion
The following example searches for the text "snacking nuts" in the `abo` index. The documents in the index contain snack descriptions in the `bullet_point` array. 
```

File: _search-plugins/search-relevance/index.md [hunk 5]
```diff
@@ -98,4 +98,60 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
...
+                "body_field": "bullet_point"
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    In the preceding query, `body_field` refers to the body field of the documents in the index, on which Kendra Intelligent Ranking ranks the results. The `body_field` is required, while the `title_field` is optional.
```
Comment (natebower) [inline _search-plugins/search-relevance/index.md:156]: Can this read "...the documents in the index, which Kendra Intelligent Ranking uses to rank the results."?

**Review by natebower — COMMENTED** [review]
@kolchfa-aws Please see my comments and changes and let me know if you have any questions. Thanks!

**Pull request merged**